uvicorn/supervisors/multiprocess.py

```diff
@@ -16,13 +16,14 @@
 
 
 class Multiprocess:
-    def __init__(self, config, target, sockets):
+    def __init__(self, config, target, sockets, workers_number):
```

Comment:
It doesn't look like we should need `workers_number` here.
In startup we're using `self.config.workers` - can we not rely on that instead? (Or am I missing something?)